src/network_analysis.jl

```diff
@@ -623,22 +623,9 @@ end
 Given the net stoichiometry matrix of a reaction system, computes a matrix of
 conservation laws, each represented as a row in the output.
 """
-function conservationlaws(nsm::T; col_order = nothing) where {T <: AbstractMatrix}
-
-    # compute the left nullspace over the integers
-    N = MT.nullspace(nsm'; col_order)
-
-    # if all coefficients for a conservation law are negative, make positive
-    for Nrow in eachcol(N)
-        all(r -> r <= 0, Nrow) && (Nrow .*= -1)
-    end
-
-    # check we haven't overflowed
-    iszero(N' * nsm) || error("Calculation of the conservation law matrix was inaccurate, "
-          * "likely due to numerical overflow. Please use a larger integer "
-          * "type like Int128 or BigInt for the net stoichiometry matrix.")
-
-    T(N')
+function conservationlaws(nsm::Matrix; col_order = nothing)
+    conslaws = positive_nullspace(nsm'; col_order = col_order)
+    Matrix(conslaws)
```

Comment:
Does `conslaws` need to be transposed again here? (Just looking at the old code).

Reply:
I think positive_nullspace does the transposition automatically in the `T(N)` line because the type we're passing in is an adjoint.